pkg/helm/charts/tests/securedclusterservices/testdata/helmtest/tls-secrets.test.yaml

```
@@ -64,3 +64,18 @@ tests:
         verifyTLSSecret("sensor")
         .secrets["collector-tls"] | assertThat(. == null)
         .secrets["admission-control-tls"] | assertThat(. == null)
+    - name: "should create TLS secrets with rhacs.redhat.com/tls=true"
+      release:
+        isInstall: true
+      values:
+        ca:
+          cert: "CA cert"
+        collector:
+          serviceTLS:
+            cert: "collector cert"
+            key: "collector key"
```

```suggestion
        collector:
          serviceTLS:
            cert: "collector cert"
            key: "collector key"
        admissionControl:
          serviceTLS:
            cert: "admission control cert"
            key: "admission control key"
```

I made two changes here:
* (optional) `ca` field not needed because it's already set for the entire suite (on line 33)
* `admissionControl` values are needed so that the `admission-control-tls` secret is created